src/main/java/springsamurais/toyswapbackend/service/member/MemberServiceImplementation.java

```diff
@@ -0,0 +1,22 @@
+package springsamurais.toyswapbackend.service.member;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import springsamurais.toyswapbackend.exception.MemberNotFoundException;
+import springsamurais.toyswapbackend.model.Member;
+import springsamurais.toyswapbackend.repository.MemberRepository;
+
+
+
+@Service
+public class MemberServiceImplementation implements MemberService {
```

Comment:
Great to have separate Member and Listing Service classes, But MemberService missing Testing

Reply:
Hi @Ainkharan the person implementing the member will do it. I created the repository and service to have a working post for testing purposes.